BrowserKit/Sources/Common/Utilities/FXFontStyles.swift

```diff
@@ -18,6 +18,7 @@ public struct FXFontStyles {
         public static let footnote = TextStyling(for: .footnote, size: 13, weight: .regular)
         public static let caption1 = TextStyling(for: .caption1, size: 12, weight: .regular)
         public static let caption2 = TextStyling(for: .caption2, size: 11, weight: .regular)
+        public static let password = TextStyling(for: .body, size: 16, weight: .regular, symbolicTraits: .traitMonoSpace)
```

Comment:
I am..... uncertain about this.  Looks like the only reason we have this in because of line 135 in `LoginDetailTableViewCell.swift` (lower in this PR). I'd argue that, instead of adding an additional font we, 1) take a look with design at whether or not that instance actually need to be monospaced, and 2) if yes, can we add that as `public static let calloutMonospaced = TextStyling(for: .body, size: 16, weight: .regular, symbolicTraits: .traitMonoSpace)` given that the rest of the stuff is the same. I just don't want to add things here without good reason / design involvement. Also, once we figure this out, we should probably add this to the Bold styles as well, given that they're supposed to be, er, mirror reflection of each other? Are they, actually?

@cyndichin you worked on these and have the best context: is this a thing we have in the design system? What would you say is the best approach

Reply:
thanks @adudenamedruby! I agree that we should follow the design system, let's confirm with @cwzilla on whether we still need monospaced for the password and if which existing text style we want to use or if we should create a new one in the system.

Reply:
Hi @cyndichin! I think it would be ideal for us to have a monospaced font for the password as desktop and Android both use it (using fonts provided by the system), and it is a better experience for users. We would probably have to create a new one in the system. Could we potentially leverage SF Mono in this use case? What was the font used for the monospace text before?

Reply:
@cwzilla thanks for reviewing, we were using Apple's system font and applying a modifier to make it monospaced. From the view debugger, it looks like it is using `.SF NS Mono`. In that case, we can keep consistent with what we were using previously. 

@NeoKolian thanks for working with us as we resolve this issue! Since we want to keep the monospace font, I was thinking we can create a new method in `TextStyling` called `monospacedFont()`. Therefore, we are aware of the distinction that this font is specifically related to monospaced and our other fonts do not need to know about this extra detail. Also, it seems passing in a weight overrides using the font descriptor.

The method I was thinking would be along the lines of:

```
      public func monospacedFont() -> UIFont {
          return DefaultDynamicFontHelper.preferredFont(withTextStyle: textStyle, size: size, symbolicTraits: [.traitMonoSpace])
      }
```

@cwzilla @adudenamedruby let me know if you have any issues with the proposal!

Reply:
@cwzilla that sounds good to me! can you give me the exact values used later? I'll add it to Figma as well :)